server/src/main/java/com/coincoinche/lobby/MatchMaker.java

```diff
@@ -0,0 +1,29 @@
+package com.coincoinche.lobby;
```

Comment:
Maybe we can put this file in `/com/coincoinche` directly if this is the only one in `/com/coincoinche/lobby`?